kerchunk/combine.py

```diff
@@ -348,10 +440,11 @@ def store_coords(self):
     def second_pass(self):
         """map every input chunk to the output"""
         # TODO: this stage cannot be rerun without clearing and rerunning store_coords too,
-        #  because some code runs dependant on the current state f self.out
+        #  because some code runs dependant on the current state of self.out
```

Comment:
dependent

Reply:
@maxrjones do you have a few minutes to review?

Reply:
@maxrjones - particularly more tests would be good, although I'm happy to merge this early and add tests as we go

Reply:
I'll have some time tomorrow afternoon to try it out

Reply:
Sorry, I got caught up with other work but this is still on my list for this week